src/app/pages/collections/messages/messages.component.ts

```diff
@@ -23,15 +23,150 @@ export class MessagesComponent implements OnInit {
+            <label>Last Name</label>
+            <input placeholder="Last Name" type="text">
+          </div>
+        </div>
+        <div suiFormField>
+          <label>Username</label>
+          <input placeholder="Username" type="text">
+        </div>
+        <div suiFormField>
+          <label>Password</label>
+          <input type="password">
+        </div>
+        <div suiFormField suiInline='true'>
+          <div class="ui checkbox">
+            <input type="checkbox" id="terms">
+            <label for="terms">I agree to the terms and conditions</label>
+          </div>
+        </div>
+        <div sui-button suiColour='blue'>Submit</div>
+      </form>
+      <div sui-message suiAttached='bottom attached' suiColour='warning'>
+        <i suiIconType="help" sui-icon></i>
+        Already signed up? <a href="#">Login here</a> instead.
+      </div>
+  `;
+
+  snippettWarning = `
+  <div sui-message suiColour='warning' [suiDismissable]="true">
+    <div sui-message-header>
+      You must register before you can do that!
+    </div>
+    <p>Visit our registration page, then try again</p>
+  </div>
+  `;
+
+  snippetInfo = `
+  <div sui-message suiColour='info' [suiDismissable]="true">
+    <div sui-message-header>
+      Was this what you wanted?
+    </div>
+    <p>It's good to see you again</p>
+  </div>
+  `;
+
+  snippetSuccess = `
+  <div sui-message suiColour='positive' [suiDismissable]="true">
+    <div sui-message-header>
+    You are eligible for a reward
+    </div>
+    <p>Go to your special offers page to see now</p>
+  </div>
+  `;
+
+  snippetError = `
+  <div sui-message suiColour='negative' [suiDismissable]="true">
+    <div sui-message-header>
+    We're sorry we can't apply that discount
+    </div>
+    <p>That offer has expired</p>
+  </div>
+  `;
+
+  snippetColored = `
+  <div sui-message suiColour='red'>Red</div>
+  <div sui-message suiColour='orange'>Orange</div>
+  <div sui-message suiColour='yellow'>Yellow</div>
+  <div sui-message suiColour='olive'>Olive</div>
+  <div sui-message suiColour='green'>Green</div>
+  <div sui-message suiColour='teal'>Teal</div>
+  <div sui-message suiColour='blue'>Blue</div>
+  <div sui-message suiColour='violet'>Violet</div>
+  <div sui-message suiColour='purple'>Purple</div>
+  <div sui-message suiColour='pink'>Pink</div>
+  <div sui-message suiColour='brown'>Brown</div>
+  <div sui-message suiColour='black'>Black</div>
+  `;
+
+  snippetSizes = `
+  <div sui-message suiSize='mini'>This is a mini message</div>
+  <div sui-message suiSize='tiny'>This is a tiny message</div>
+  <div sui-message suiSize='small'>This is a small message</div>
+  <div sui-message suiSize='large'>This is a large message</div>
+  <div sui-message suiSize='big'>This is a big message</div>
+  <div sui-message suiSize='huge'>This is a huge message</div>
+  <div sui-message suiSize='massive'>This is a massive message</div>
+  `;
+
+  constructor() {}
 
   toggleStateMap: any = {};
```

Comment:
@bolorundurovj I dont think we still need the toggle state map. Do we?

Reply:
We do not. I'll Remove it